### PR TITLE
Cxtb 145 click and view participant record from listing

### PR DESCRIPTION
**Pull request by cmoreno-nts**
Automated test case:
[https://digitalscientists.atlassian.net/browse/CXTB-256](url)

Successful run image:
image

### CODE REVIEW

**Review by csanchez-nts — DISMISSED** [review]

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml [hunk 1]
```diff
@@ -290,6 +290,85 @@ TestOnboarderAccessParticipantDirectoryandFilter:
       Id: $PAGE.care_platform_participant.status_dropdown_options$
     - Type: click
       Strategy: xpath
-      Id: $PAGE.care_platform_participant.status_dropdown_options$
-      
+      Id: $PAGE.care_platform_participant.status_dropdown_options$      
 
+# CXTB-256: Click and View Participant Record from Listing
```
Comment (dguzmanneural) [inline tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml:295]: We usually use the TC number here.
I believe the one corresponding to this is CXTB-145

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml [hunk 2]
```diff
@@ -290,6 +290,85 @@ TestOnboarderAccessParticipantDirectoryandFilter:
...
+        - var: SUBTAB_CLASS
+          attr: class
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Asserts:
+        - Type: contain
+          Var: SUBTAB_CLASS
+          Value: 'active'
+  # Click on Behavioral Subtask
```
Comment (dguzmanneural) [inline tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml:341]: is it subtask or subtab?

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml [hunk 3]
```diff
@@ -290,6 +290,85 @@ TestOnboarderAccessParticipantDirectoryandFilter:
...
+        - var: SUBTAB_CLASS
+          attr: class
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Asserts:
+        - Type: contain
+          Var: SUBTAB_CLASS
+          Value: 'active'
+  # Click on Medical History Subtask
```
Comment (dguzmanneural) [inline tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml:358]: Subtask?

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml (hunk 2 above)
Comment (cmoreno-nts) [reply tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml]: SubTab, just fixed.

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml (hunk 1 above)
Comment (cmoreno-nts) [reply tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml]: already fixed.

**Review by csanchez-nts — APPROVED** [review]

**Review by dguzmanneural — APPROVED** [review]

**Review by miguel-hquez — APPROVED** [review]

**Pull request merged**